src/Haskell/Ide/Engine/Plugin/Base.hs

```diff
@@ -110,7 +111,7 @@ getProjectGhcVersion :: BIOS.Cradle -> IO String
 getProjectGhcVersion crdl = do
   isStackProject   <- doesFileExist "stack.yaml"
   isStackInstalled <- isJust <$> findExecutable "stack"
-  if BIOS.actionName (BIOS.cradleOptsProg crdl) == "stack" && isStackProject && isStackInstalled
+  if isStackCradle crdl && isStackProject && isStackInstalled
```

Comment:
Don't need `isStackProject` here anymore?

Reply:
True, that is redundant now. The `isStackInstalled` should also be resolved somewhere else, in my opinion!